### ISSUE TITLE
Port number is not stored in the list of known hosts

### ISSUE THREAD

**Issue by marcelflymark** [luandersonn/MyFTP#51]
Hello Mr. Airton,

When I try to connect to my server via the known hosts button, the port is fixed to 21.

Could you please store the port number as well?

(I use a number different than 21 to reduce log attempts by bad actors.
Also, less log entries.)

Ty,
Marcel

**Comment by luandersonn** [comment]
Absolutely. I'm working on it in the WinUI update. Please, check out the issue #45
